src/components/SearchPanelControls.js

```diff
@@ -52,46 +85,80 @@ export class SearchPanelControls extends Component {
 
   /** */
   render() {
-    const { companionWindowId, t } = this.props;
+    const {
+      companionWindowId, t, searchHits, selectedContentSearchAnnotation, classes,
+    } = this.props;
+
+    const currentHitIndex = searchHits
+      .findIndex(val => val.annotations.includes(selectedContentSearchAnnotation[0]));
     const { search } = this.state;
     const id = `search-${companionWindowId}`;
     return (
-      <form onSubmit={this.submitSearch}>
-        <FormControl>
-          <InputLabel htmlFor={id}>{t('searchInputLabel')}</InputLabel>
-          <Input
-            id={id}
-            onChange={this.handleChange}
-            value={search}
-            endAdornment={(
-              <InputAdornment position="end">
-                <IconButton
-                  aria-label={t('searchSubmitAria')}
-                  type="submit"
-                >
-                  <SearchIcon />
-                </IconButton>
-              </InputAdornment>
-            )}
-          />
-        </FormControl>
-      </form>
+      <>
+        <form onSubmit={this.submitSearch}>
+          <FormControl>
+            <InputLabel htmlFor={id}>{t('searchInputLabel')}</InputLabel>
+            <Input
+              id={id}
+              onChange={this.handleChange}
+              value={search}
+              endAdornment={(
+                <InputAdornment position="end">
+                  <IconButton
+                    aria-label={t('searchSubmitAria')}
+                    type="submit"
+                  >
+                    <SearchIcon />
+                  </IconButton>
+                </InputAdornment>
+              )}
+            />
+          </FormControl>
+        </form>
+        {(searchHits.length > 0) && (
+          <Typography variant="body2" align="center" classes={classes}>
+            <MiradorMenuButton
+              aria-label={t('searchPreviousResult')}
+              disabled={!this.hasPreviousResult(currentHitIndex)}
+              onClick={() => this.previousSearchResult(currentHitIndex)}
+            >
+              <ChevronLeftIcon />
+            </MiradorMenuButton>
+              {currentHitIndex + 1}
```

Comment:
This string should be i18n'ed, yes?

Reply:
Most definitely. I've pushed this up with a rebase from autocomplete.